libs/dh/charges/feature-charges/src/components/charge.component.ts

```diff
@@ -49,26 +54,63 @@ import { TranslocoDirective } from '@jsverse/transloco';
       overflow: auto;
     }
   `,
-  imports: [TranslocoDirective, VaterStackComponent, VaterSpacerComponent, WATT_LINK_TABS],
+  imports: [
+    TranslocoPipe,
+    TranslocoDirective,
+    VaterStackComponent,
+    VaterSpacerComponent,
+    WATT_LINK_TABS,
+    DhEmDashFallbackPipe,
+    DhChargeStatusComponent,
+    DhChargeActionsComponent,
+  ],
   template: `<div class="page-grid">
     <div class="page-header" vater-stack direction="row" gap="m" wrap align="end">
       <div *transloco="let t; prefix: 'charges.charge'">
-        <h2 vater-stack direction="row" gap="m" class="watt-space-stack-s">headline</h2>
+        <h2 vater-stack direction="row" gap="m" class="watt-space-stack-s">
+          {{ charge()?.chargeId | dhEmDashFallback }} {{ charge()?.chargeName | dhEmDashFallback }}
+          @let status = charge()?.status;
+
+          @if (status) {
+            <dh-charge-status [status]="status" />
+          }
+        </h2>
 
         <vater-stack direction="row" gap="ml">
-          <span class="watt-text-s"> info </span>
+          <span class="watt-text-s">
+            <span class="watt-label watt-space-inline-xs">{{ t('type') }}</span>
+            @let chargeType = charge()?.chargeType;
+            @if (chargeType) {
+              {{ 'charges.chargeTypes.' + chargeType | transloco }}
+            }
+          </span>
+
+          <span direction="row" gap="s" class="watt-text-s">
+            <span class="watt-label watt-space-inline-xs">{{ t('owner') }}</span>
+
+            {{ charge()?.chargeOwnerName | dhEmDashFallback }}
+          </span>
 
-          <span direction="row" gap="s" class="watt-text-s"> info </span>
+          <span direction="row" gap="s" class="watt-text-s">
+            <span class="watt-label watt-space-inline-xs">{{ t('resolution') }}</span>
+            {{ 'charges.resolutions.' + charge()?.resolution | transloco }}
+          </span>
 
-          <span direction="row" gap="s" class="watt-text-s"> info </span>
+          <span direction="row" gap="s" class="watt-text-s">
+            <span class="watt-label watt-space-inline-xs">{{ t('vat') }}</span>
+            {{ 'charges.vatClassifications.' + charge()?.vatClassification | transloco }}
+          </span>
 
-          <span direction="row" gap="s" class="watt-text-s"> info </span>
+          <span direction="row" gap="s" class="watt-text-s">
+            <span class="watt-label watt-space-inline-xs">{{ t('transparentInvoicing') }}</span>
+            {{ charge()?.transparentInvoicing ? ('yes' | transloco) : ('no' | transloco) }}
+          </span>
```

Comment:
suggestion: since this is a near duplicate of the metering point header, maybe we should consider creating a shared component for this fx. `dh-header`, that can project headlines, watt-description-list, actions etc. 

Just a suggestion for a future PR 🙂